x-pack/plugins/index_management/common/constants/home_sections.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export enum Section {
```

Comment:
We need to extract these constants to a separate file in common folder so that it could be imported without increasing the bundle size.